app/controllers/simplified-account/settings/team-members/team-members.controller.test.js

```diff
@@ -1,104 +1,88 @@
-      expect(responseStub.args[0]).to.include(req)
-      expect(responseStub.args[0]).to.include(res)
-      expect(responseStub.args[0]).to.include('simplified-account/settings/team-members/index')
+      expect(mockResponse.args[0][0].user).to.deep.include(adminUser)
+      expect(mockResponse.args[0]).to.include(res)
+      expect(mockResponse.args[0]).to.include('simplified-account/settings/team-members/index')
     })
 
     it('should pass context data to the response method', () => {
-      expect(responseStub.args[0][3]).to.have.property('teamMembers').to.have.property('admin').to.have.length(1)
-      expect(responseStub.args[0][3].teamMembers.admin[0]).to.have.property('externalId').to.equal('user-id-for-admin-user')
-      expect(responseStub.args[0][3].teamMembers).to.have.property('view-only').to.have.length(1)
-      expect(responseStub.args[0][3].teamMembers['view-only'][0]).to.have.property('externalId').to.equal('user-id-for-view-only-user')
+      expect(mockResponse.args[0][3]).to.have.property('teamMembers').to.have.property('admin').to.have.length(1)
+      expect(mockResponse.args[0][3].teamMembers.admin[0]).to.have.property('externalId').to.equal('user-id-for-admin-user')
+      expect(mockResponse.args[0][3].teamMembers).to.have.property('view-only').to.have.length(1)
+      expect(mockResponse.args[0][3].teamMembers['view-only'][0]).to.have.property('externalId').to.equal('user-id-for-view-only-user')
 
-      expect(responseStub.args[0][3]).to.have.property('invitedTeamMembers').to.have.property('admin').to.have.length(1)
-      expect(responseStub.args[0][3].invitedTeamMembers.admin[0]).to.have.property('email').to.equal('invited-admin-user@user.gov.uk')
-      expect(responseStub.args[0][3].invitedTeamMembers).to.have.property('view-only').to.have.length(1)
-      expect(responseStub.args[0][3].invitedTeamMembers['view-only'][0]).to.have.property('email').to.equal('invited-view-only-user@user.gov.uk')
+      expect(mockResponse.args[0][3]).to.have.property('invitedTeamMembers').to.have.property('admin').to.have.length(1)
+      expect(mockResponse.args[0][3].invitedTeamMembers.admin[0]).to.have.property('email').to.equal('invited-admin-user@user.gov.uk')
+      expect(mockResponse.args[0][3].invitedTeamMembers).to.have.property('view-only').to.have.length(1)
+      expect(mockResponse.args[0][3].invitedTeamMembers['view-only'][0]).to.have.property('email').to.equal('invited-view-only-user@user.gov.uk')
 
-      expect(responseStub.args[0][3]).to.have.property('inviteTeamMemberLink')
+      expect(mockResponse.args[0][3]).to.have.property('inviteTeamMemberLink')
         .to.equal('/simplified/service/service-id-123abc/account/test/settings/team-members/invite')
-      expect(responseStub.args[0][3]).to.have.property('numberInvitedMembers').to.equal(2)
+      expect(mockResponse.args[0][3]).to.have.property('numberInvitedMembers').to.equal(2)
```

Comment:
Should we test for the properties `isServiceAdmin` and `roles` here too?

Reply:
good point, done.